### DISCUSSION TITLE
Can the kbd shortcuts be made prettier?

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#5876] [Feature Requests]
### Description

The `kbd` shortcuts that Quarto started providing post its v1.3, don't currently seem very pretty and the different symbols look rather out of proportion with bad spacing. For instance, following is a comparison with those in the Arc web browse's Settings:

1. Arc
<img width="334" alt="Screenshot 2023-06-11 at 5 53 30 PM" src="https://github.com/quarto-dev/quarto-cli/assets/102873841/ed320264-ec71-4c73-acdd-39b37f12cb5c">

2. Quarto
<img width="814" alt="Screenshot 2023-06-11 at 5 53 59 PM" src="https://github.com/quarto-dev/quarto-cli/assets/102873841/4130ea85-213d-4026-b9cd-2fc89696d019">

Can we make the Quarto rendition more aesthetic?

**Comment by mcanouil** [comment]
This depends on the font, especially in this case it uses the monospace font (maybe it should not).

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr>
<td>

````qmd
---
title: "Quarto Playground"
format:
  html:
    include-in-header:
      - text: |
          <style>
          kbd {
            font-family: var(--bs-font-sans-serif) !important;
          }
          </style>
---

This is a playground for Quarto.

To print, press {{< kbd Shift-Ctrl-P >}}.
To open an existing new project,
press {{< kbd mac=Shift-Command-O win=Shift-Control-O linux=Shift-Ctrl-L >}}.

````

</td>
<td>
<img width="386" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/4d6f525b-372a-4f84-a828-9a56a5d782cd">
</td>
</tr>
</table>

**Reply by mcanouil** [reply]
I created an issue and made a proposal to Quarto team, see #5880

Closing this thread in favour of the issue.
